### PR TITLE
Revising steps for building MXNet on Windows platform 

### PR DESCRIPTION
**Pull request by eftiquar**
## Description ##
The steps for building MXNet on Windows have been enhanced by adding more details to make it more customer friendly. The GUI version of CMake simplifies generation of Visual Studio project by iteratively 
providing feedback on generation errors. Also by using appropriate Visual Studio Command prompt, CMake tool will inherit all the details it needs to know to discover build environment.

### CODE REVIEW

**Comment by yajiedesign** [comment]
There are problems with vs2017 +cuda9.0 now.some cu can't build.

**Comment by yajiedesign** [comment]
ok,vs 2017 and cuda9.0 compiler problem has been found.the default `All` use too many arc.cause ptxas is carsh.cut down arc it will work fine.
https://github.com/apache/incubator-mxnet/pull/8624

File: docs/install/windows_setup.md [hunk 1]
```diff
@@ -33,9 +34,19 @@ To build and install MXNet yourself, you need the following dependencies. Instal
 After you have installed all of the required dependencies, build the MXNet source code:
 
 1. Download the MXNet source code from [GitHub](https://github.com/dmlc/mxnet).
-2. Use [CMake](https://cmake.org/) to create a Visual Studio solution in ```./build```.
-3. In Visual Studio, open the solution file,```.sln```, and compile it.
-These commands produce a library called ```mxnet.dll``` in the ```./build/Release/``` or ```./build/Debug``` folder.
+2. Open native tools command prompt for Visual Studio 2015 ( choose x64 or x86 native as required)
```
Comment (yajiedesign) [inline docs/install/windows_setup.md]: There's no need run cmake gui in tools command prompt.

Comment (eftiquar) [reply docs/install/windows_setup.md]: Not doing so results in errors from CUI being unable to discover VS tools.

Comment (yajiedesign) [reply docs/install/windows_setup.md]: no.If there's no problem in your system.gui cmake can find vs with direct run.

**Comment by piiswrong** [comment]
Closing due to inactive.
@yajiedesign Please reopen if necessary.

**Pull request closed without merging**